Cargo.toml

```diff
@@ -135,8 +135,8 @@ webpki-roots = "0.25"
 webrender = { git = "https://github.com/servo/webrender", branch = "0.64", features = ["capture"] }
 webrender_api = { git = "https://github.com/servo/webrender", branch = "0.64" }
 webrender_traits = { path = "components/shared/webrender" }
-wgpu-core = { git = "https://github.com/gfx-rs/wgpu", rev = "d0a5e48aa7e84683114c3870051cc414ae92ac03" }
-wgpu-types = { git = "https://github.com/gfx-rs/wgpu", rev = "d0a5e48aa7e84683114c3870051cc414ae92ac03" }
+wgpu-core = { git = "https://github.com/gfx-rs/wgpu", rev = "1e784c9c0af7a785b62e3b6840ed012a7477520f" }
+wgpu-types = { git = "https://github.com/gfx-rs/wgpu", rev = "1e784c9c0af7a785b62e3b6840ed012a7477520f" }
```

Comment:
Is it possible to switch to a release of wgpu instead of using a git hash?

Reply:
No, unfortunately not. Releases are cut too slowly and last ones had some nasty bugs. Also I do some work on wgpu and keeping servo up to date makes it easier to squash bugs in servo and wgpu at the same time.

BTW firefox also uses git rev instead of releases updated weekly, for similar reasons.